### PR TITLE
send email after creating server

### PR DESCRIPTION
**Pull request by rafiquecudjoe**

### CODE REVIEW

File: backend/src/database/migrations/20221114095422_create-server.js [hunk 1]
```diff
@@ -3,6 +3,7 @@ export async function up(knex) {
     table.uuid('id').primary().defaultTo(knex.raw('(UUID())'));
     table.string('name', 255).notNullable();
     table.string('ipAddress', 255).notNullable();
+    table.string('email', 255).notNullable();
```
Comment (orlando70) [inline backend/src/database/migrations/20221114095422_create-server.js:6]: Remove this.

File: backend/src/controllers/ServerController.js [hunk 2]
```diff
@@ -20,6 +21,9 @@ export default class ServerController {
             const {id} = req.session.user;
```
Comment (orlando70) [inline backend/src/controllers/ServerController.js:21]: You can easily destructure user email from here. Why getting it from request body?

**Review by orlando70 — CHANGES_REQUESTED** [review]

File: backend/src/database/migrations/20221114095422_create-server.js (hunk 1 above)
Comment (Usenmfon) [reply backend/src/database/migrations/20221114095422_create-server.js]: Changes have been made, email field removed.

**Review by Usenmfon — APPROVED** [review]
changes made and approved.

**Review by Usenmfon — APPROVED** [review]

**Pull request merged**